### PR TITLE
marking bar code with beta tag

### PR DESCRIPTION
**Pull request by lakhveerk**
marking bar code with beta tag for potential error fixes or changes since this is a new capability

### CODE REVIEW

File: change/@microsoft-teams-js-3e90cf0a-ad19-48b6-aa04-d66b3595072e.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "marked barCode capability beta for potential future changes",
```
Comment (erinha) [inline change/@microsoft-teams-js-3e90cf0a-ad19-48b6-aa04-d66b3595072e.json:3]: Please change this to `Added beta tag to `barCode` capability to accurately reflect level of support offered`

Comment (erinha) [reply change/@microsoft-teams-js-3e90cf0a-ad19-48b6-aa04-d66b3595072e.json:3]: Ugh, backticks around `barCode`, not the rest :)

File: change/@microsoft-teams-js-3e90cf0a-ad19-48b6-aa04-d66b3595072e.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added beta tag to barCode capability to accurately reflect level of support offered",
```
Comment (erinha) [inline change/@microsoft-teams-js-3e90cf0a-ad19-48b6-aa04-d66b3595072e.json:3]: Can you please add backticks to "barCode"?

Comment (lakhveerk) [reply change/@microsoft-teams-js-3e90cf0a-ad19-48b6-aa04-d66b3595072e.json]: haha just pushed

**Review by erinha — APPROVED** [review]

**Review by jadahiya-MSFT — APPROVED** [review]

**Pull request merged**